index.js

```diff
@@ -1,10 +1,13 @@
 import {getProperty} from 'dot-prop';
 
-export default function sortOn(array, property) {
+export default function sortOn(array, property, options) {
 	if (!Array.isArray(array)) {
 		throw new TypeError(`Expected type \`Array\`, got \`${typeof array}\``);
 	}
 
+	const locales = (options && options.locales) || undefined;
+	const localeOptions = (options && options.localeOptions) || undefined;
```

Comment:
Not sure if you prefer some other checks here? If this was targeting 14+ I would have used null coalescing, but since we target Node 12 I settled for this.